content/en/users/data/management/datahub/_index.md

```diff
@@ -0,0 +1,118 @@
+---
```

Comment:
This one is way larger than other services, feels a bit unbalanced. Are we expecting other services to have an overview page that grow this much?

Reply:
All should be like that. With details in subpages, neatly broken up by subject. So let's keep this, the others we'll improve in future PRs.